libs/chain-signatures/contract/src/tee/tee_state.rs

```diff
@@ -30,31 +31,59 @@ impl Default for TeeState {
             allowed_docker_image_hashes: Default::default(),
             historical_docker_image_hashes: Default::default(),
             votes: Default::default(),
+
             tee_participant_info: IterableMap::new(StorageKey::TeeParticipantInfo),
         }
     }
 }
-
 impl TeeState {
+    /// A shared helper method to verify the TEE quote and Docker image.
+    /// This method returns a `Result` with the `TeeQuoteStatus` or an `Error`.
```

Comment:
Nit: last line is redundant, the return type is visible in the signature.

```suggestion
    /// Verifies the TEE quote and Docker image.
```